assets/schema_input.json

```diff
@@ -40,10 +40,20 @@
                 "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
             },
             "rundir": {
-                "type": "string",
-                "format": "path",
-                "exists": true,
-                "errorMessage": "Run directory must be a path",
+                "oneOf": [
+                    {
+                        "type": "string",
+                        "format": "file-path",
+                        "exists": true,
+                        "pattern": "^\\S+\\.tar\\.gz$"
```

Comment:
I wonder, if people will be annoyed if we exclude whitespace characters? 

Evidently not good practise, but people seem to like names like `/home/user/thesis/data/project 01/sequencing/raw/chip-seq samples/`, at least I see them once in a while in the error logs posted on Slack. So I think we must anyway ensure that the pipeline used properly quoted arguments everywhere.

Reply:
I would prefer not to allow it, but it might be a lot of work to get a user to restructure their project just to use a workflow. And usability trumps my opinion on better practice.

Reply:
Frankly, it will mostly be NGI using the workflow, so we should make sure it suits our needs. However, I do not see a true downside for us to strictly handle everything quoted, the extra effort should be minimal (_unless there are some sloppy modules out there that would need to be patched?_)

We can still replace white spaces with underscores in every file process output that the pipeline produces?

Reply:
I think it's good practice to user-proof modules anyway.